app/controllers/intercom_controller.rb

```diff
@@ -93,6 +94,21 @@ def save_and_render_configuration # rubocop:disable Metrics/MethodLength
     })
   end
 
+  def create_and_sign_in_user(intercom_data, company)
+    @user = User.new
+    @user.email = intercom_data.email
+    @user.first_name = intercom_data.name.split[0]
+    @user.last_name = intercom_data.name.split[-1]
```

Comment:
```suggestion
    @user.name = intercom_data.name
```

We have a method that does this for us: https://github.com/pranav7/cadet-app/blob/e74ab3843c7e965c1d999c834c2b136dd26ed513/app/models/user.rb#L68-L70

Reply:
Updated the code. Since `self.first_name` and `self.last_name` are set, I'm assuming they will be saved to the database when we do `@user.save` right?